version.h

```diff
@@ -1,6 +1,6 @@
 /* $OpenBSD: version.h,v 1.80 2017/09/30 22:26:33 djm Exp $ */
 
-#define SSH_VERSION	"OpenSSH_7.6"
+#define SSH_VERSION	"OpenSSH_for_Windows_7.6"
```

Comment:
since this is not specific to windows, we should have #ifdef windows?

Reply:
I specifically did not because of 2 reasons:
1. These definitions are in a very light header and it would be very likely that this header gets included before "config.h" that actually #defines "Windows"
2. Keep it this way would give us an explicit merge conflict while integrating upstream changes  that modify SSH_VERSION. We are bound to then pay attention and make the appropriate version changes, as needed on our side (vs accidentally ignoring upstream version changes and making our version stale).